src/20231108_AaveV3Ethereum_AddFXSToEthereumV3/AddFXSToEthereumV3.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Add FXS to Ethereum V3"
+author: "Alice Rozengarden (@Rozengarden - Aave-chan initiative)"
+discussions: "https://governance.aave.com/t/arfc-add-fxs-to-ethereum-v3/15112"
+---
+
+## Simple Summary
+
+This publication presents the community an opportunity to add FXS to the Ethereum Aave v3 Liquidity Pool.
+
+## Motivation
+
+Frax Finance is a DeFi protocol that offers a stable protocol for issuing FRAX/FPI/frxETH and subprotocols such as Fraxlend, Fraxswap, and Fraxferry. The Frax Shares Token ($FXS) serves a dual role as both a governance token and a utility token.
+
+The addition of FXS to Aave V3 represents the introduction of a new asset, encouraging the transition from Aave V2 to V3 and diversifying collateral assets for GHO. Furthermore, Aave DAO can utilize FXS obtained from the pool to kickstart the adoption of GHO and/or new aTokens.
+
+## Specification
+
+The parameters shown below are supported by @Gauntlet
+Ticker: FXS
+Contract Adress: 0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0
```

Comment:
better put an ehterscan link for addresses `[0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0](etherscan link)`